src/ray/core_worker/core_worker_options.h

```diff
@@ -146,6 +147,9 @@ struct CoreWorkerOptions {
   /// any long-running operations in the core worker will short circuit and return that
   /// status.
   std::function<Status()> check_signals;
+  /// Language callback that initializes a thread and returns a function to be called
```

Comment:
```suggestion
  /// Application-language callback that initializes a thread and returns a function to be called
```
(matching formatting of the below fields)

Reply:
Updated: https://github.com/ray-project/ray/pull/50644/commits/620b7973dab49e35bf794744565b48a6d39fe40f